index.d.ts

```diff
@@ -0,0 +1,22 @@
+import { NightwatchAPI, Element } from 'nightwatch';
+
+declare module 'nightwatch' {
+  interface NightwatchAPI {
+    importScript(
+      scriptPath: string,
+      options: { scriptType: string; componentType: string },
+      callback?: () => void
+    ): this;
+    mountReactComponent<TProps extends Record<string, any>>(
```

Comment:
this isn't exported by this plugin, it's only the `mountComponent`.

Reply:
Ah! thanks for pointing out, removed